po4a-create-all-targets.sh

```diff
@@ -94,7 +94,7 @@ use_po_module () {
 while IFS= read -r -d '' dir
 do
 	lang=$(basename -s .md "$dir")
-	echo "$lang"
+	echo '$lang'
```

Comment:
Wait. I think you need double quotes here to allow the content of variables be shown not the string $lang

Reply:
Ah yes, I got a bit overenthusiastic replacing the double quotes ...

Reply:
Hmmm, I'm sure I tested it before submitting though... I'll have a look tomorrow, can't check now.

Reply:
Well, no quotes at all are required as it's just `echo` - but definitely not single quotes...